app/models/game.rb

```diff
@@ -193,7 +210,8 @@ def populate_board!
       row_coordinate: 7,
       column_coordinate: 4,
       color: 'black',
-      image: 'pieces/BlackKing.png'
+      image: 'pieces/BlackKing.png',
+      is_on_board?: true
```

Comment:
We could add a database default so all new pieces are "on the board" when created.